src/Graph.h

```diff
@@ -159,5 +195,9 @@ class Graph
      */
     bool visited(Node* node, std::vector<Node*> vec);
     
-    std::vector<Node*> nodes_;    
+    std::vector<Node*> nodes_;
+    // below are betweenness private variables
+    std::vector<std::pair<Node*, Node*>> node_pairs;
+    std::vector<std::tuple<double, double, double>> relationships_between_nodes;
```

Comment:
guys... im lowkey cringing at the snake case... but if you guys want it I guess but not a fan